ida-ws/src/test/resources/scikit/datadump/cluster_datadump.json

```diff
@@ -0,0 +1,402 @@
+{

```

Comment:
Delete this file. We don't use the old scrape format anymore.

Reply:
deleted it